### DISCUSSION TITLE
Is there a repo with source code for just available?

### DISCUSSION THREAD

**Discussion by jirkavrba** [maciejwalkowiak/just#40] [Q&A] [answered]
Thanks for this amazing tool, been using it since the release.

I was just wondering if there's another repository with sources, because I'm really interested in how's the whole thing built.

I wasn't able to find links here or on the project website, so I'm assuming the source code is not available, but it's still worth a try to ask here :slightly_smiling_face:

**Comment by maciejwalkowiak** [comment, marked as answer]
Thank you for kind words! I'm glad you like it. Just is not open source though. Hope it's not a deal breaker for you!

**Reply by jirkavrba** [reply]
Thanks for the info. It's not a deal breaker for sure. I was just curious :slightly_smiling_face:
